examples/rp/src/bin/bluetooth.rs

```diff
@@ -14,7 +14,7 @@ use embassy_rp::bind_interrupts;
 use embassy_rp::gpio::{Level, Output};
 use embassy_rp::peripherals::{DMA_CH0, PIO0};
 use embassy_rp::pio::{InterruptHandler, Pio};
-use embassy_sync::blocking_mutex::raw::NoopRawMutex;
+use embassy_sync_06::blocking_mutex::raw::NoopRawMutex as NoopRawMutex06;
```

Comment:
Let it be known that I'm not trying to hide this, but I also can't do anything against this :(

Reply:
we should remove the example in this repo, to break the circular dep. examples in the trouble repo are more maintained.